src/lib.rs

```diff
@@ -1449,137 +1397,10 @@ fn is_valid_host(host: &str) -> bool {
         }
     }
 }
-
```

Comment:
We still need this complexity for Gaggles, as in that case each Goose Worker has a parent thread tracking statistics then pushes them to the Gaggle Manager where they have to merged. It's just moving to another file as it's not needed here anymore.